udata/core/organization/views.py

```diff
@@ -291,6 +296,26 @@ def datasets_csv(org):
     return csv.stream(adapter, '{0}-datasets'.format(org.slug))
 
 
+@blueprint.route('/<org:org>/issues.csv')
+def issues_csv(org):
+    datasets = Dataset.objects.filter(organization=str(org.id))
+    issues = [DatasetIssue.objects.filter(subject=dataset)
```

Comment:
After trying a few things, it appears that the planned `ReferenceField` refactoring is necessary to get rid of the manual list comprehension. I'll keep it as-is for now.